modulefiles/GDAS/hera.lua

```diff
@@ -63,6 +63,7 @@ load("ncview/2.1.8")
 load("netcdf-cxx4/4.3.1")
 load("json/3.10.5")
 load("py-pybind11/2.8.1")
+--load("crtm/v2.4_jedi")
```

Comment:
Apart from the endianness, is `crtm/v2.4_jedi` consistent with WCOSS2 `crtm/2.4.0` which goes operational with GFS v16.3.3?

Reply:
great question, and I'm not sure. I would hope so but to be honest I wouldn't wager a large amount of my daily double on it.

Reply:
OK.  I guess we'll cross this bridge when we get to it.